static/js/breakpoint-handler.js

```diff
@@ -22,18 +24,18 @@ function getViewportDimensions() {
  *  enter: callback when breakpoint region entered,
  *  leave: callback when breakpoint region exited
  * }
- * 
+ *
  */
 
-BreakpointHandler = function (opts) {
-    opts || (opts = {});
+function BreakpointHandler(opts) {
```

Comment:
I'd set the `no-undef` [rule](http://eslint.org/docs/rules/no-undef.html) which means since this was in the global scope (note no `var`), it would need a `/*global */` designator. Function declarations are preferable IMO because they get hoisted so there's less likelihood of calling a function before it's defined.

However, good catch, as I realized this is being called in a global context elsewhere. Fixed in https://github.com/cfpb/cfgov-refresh/commit/f02268bd02e73edb75ee96fa24c7b8b5f391fdda.

This all needs to be refactored into modules per https://github.com/cfpb/cfgov-refresh/issues/244